PopPUNK/assign.py

```diff
@@ -475,12 +504,18 @@ def main():
                  args.plot_fit,
                  args.graph_weights,
                  args.max_a_dist,
+                 args.max_pi_dist,
+                 args.reference_isolate,
```

Comment:
The call to assign in web.py needs to be updated to add these arguments

Reply:
Updated in c7daea2.

Reply:
And 090994f.